### ISSUE TITLE
[Alerts] Inoperative notifications when any feedback event accepted

### ISSUE THREAD

**Issue by JLuse** [getsentry/sentry#71793]
### Environment

SaaS (https://sentry.io/)

### Steps to Reproduce

**Description**
It has been reported that alert rules, intended to trigger when the IF condition `The issue's category is equal to Feedback` is met, are not activating. This is happening despite new feedback events occurring.

Example alert rule and various feedback issues as internal note in support ticket: https://sentry.zendesk.com/agent/tickets/122931

### Expected Result

Notifications to be sent for any occur of User Feedback 

### Actual Result

Alert rule not triggered

### Product Area

Alerts

**Comment by michellewzhang** [comment]
@jas-kas i don't have access to zendesk -- could you summarize the customer's issue and whether the issue is still ongoing?

**Comment by billyvg** [comment]
This is more of a misunderstanding from the user (as well as some confusing behaviors on our end).

Crash-modal alerts are not controlled by our Alert rules, but rather in Project Settings

![Image](https://github.com/user-attachments/assets/a83c7e12-d6cb-419e-bda6-b0222b8e1302)

**Comment by michellewzhang** [comment]
ah perfect, we added that blue banner recently so hopefully that clarifies things for future users
